src/services/GeojsonService.js

```diff
@@ -110,20 +148,31 @@ export default class GeojsonMapService {
         const { name, gender } = e.features[0].properties;
         const color = gender === FEMALE ? '#ffca3af2' : '#0e9686f2';
         const popupType = gender === FEMALE ? 'popup-female' : 'popup-male';
+        const getHTML = this.getHTMLWikipediaLink;
 
-        const html = `<div class="row"><div class="col-sm"><div class="${popupType}"><p>${name}</p>
-          ${gender === FEMALE ? this.getHTMLWikipediaLink(link, popupText) : ''}</div></div></div>`;
+        // eslint-disable-next-line no-console
+        getWikdataDetails(link).then((data) => {
```

Comment:
Same as before, WDYT about re-naming this `data` to something a bit more explicative? 

And just to be sure, `getWikdataDetails` is a typo, or did you mean `getWikidataDetails`?